modules/OCPPConfiguration/main/util.hpp

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#pragma once
+
+#include "generated/types/ocpp.hpp"
+#include "mapping_reader.hpp"
+#include "utils/yaml_loader.hpp"
+#include <c4/yml/std/map.hpp>
+#include <filesystem>
+#include <ryml.hpp>
+#include <sstream>
+
+namespace module {
+namespace Util {
```

Comment:
Please use lower case for namespace e.g. `util`

Reply:
Done